readthedocs/projects/forms.py

```diff
@@ -201,6 +195,60 @@ class Meta(object):
             # Version Support
             # 'num_major', 'num_minor', 'num_point',
         )
+        initial = {'analytics_code': 'foo'}
+
+    def __init__(self, *args, **kwargs):
+        super(ProjectAdvancedForm, self).__init__(*args, **kwargs)
+
+        # default_version ChoiceField
+        versions_qs = self.instance.versions.all()
+        active = versions_qs.filter(active=True)
+
+        if active.exists():
+            version_choices = [(version.slug, version.verbose_name) for version in active]
+            version_choices = version_choices + [(LATEST, None)]
+            self.fields['default_version'].widget = forms.Select(choices=version_choices)
+
+        # default_branch ChoiceField
+        branches = self.instance.vcs_repo().branches
+        branch_choices = [(branch.verbose_name, branch.verbose_name) for branch in branches]
+        branch_choices = [(None, None)] + branch_choices
+        self.fields['default_branch'].widget = forms.Select(choices=branch_choices)
+
+        # configuration file ChoiceField
+        conf_file_choices = self.give_file_choices('conf.py')
+        if(not conf_file_choices):
```

Comment:
nitpick: spacing is off on `if(`, it should be `if (`. Also, the parens are not required. Running through autolinting should resolve these issues.